src/components/icon/icon.component.js

```diff
@@ -36,6 +39,13 @@ const Icon = React.forwardRef(
     },
     ref
   ) => {
+    if (!deprecatedWarnTriggered) {
```

Comment:
I think we should scope these to only warn if either the props has been used to avoid spamming the console with warnings unnecessarily 
```suggestion
    if (!deprecatedWarnTriggered && (iconColor || bgThme)) {
```

Reply:
Good point, changed, thanks